src/block/video-popup/index.php

```diff
@@ -19,3 +19,60 @@ function stackable_load_videopopup_frontend_script() {
 	}
 	add_action( 'stackable/video-popup/enqueue_scripts', 'stackable_load_videopopup_frontend_script' );
 }
+
+if ( ! class_exists( 'Stackable_Video_Popup_Schema' ) ) {
+	class Stackable_Video_Popup_Schema {
+		public $video_entities = [];
+
+		function __construct() {
+			add_filter( 'render_block_stackable/video-popup', array( $this, 'render_block_video_popup_schema' ), 10, 2 );
+			add_filter( 'wp_footer', array( $this, 'print_video_popup_schema' ) );
+		}
+
+		public function print_video_popup_schema() {
+			if ( count( $this->video_entities ) ) {
+				// Compile all video schema entities into a single script
+				echo '<script type="application/ld+json"> [ ' . implode( ', ', $this->video_entities ) . ' ] </script>';
+			}
+		}
+
+		public function render_block_video_popup_schema( $block_content, $block ) {
+			// Initialize video schema
+			$video_schema = array();
+			$video_schema[ '@context' ] = 'https://schema.org';
+			$video_schema[ '@type' ] = 'VideoObject';
```

Comment:
You can initialize `$video_schema` with the default array values here right away instead of doing 2 additional assignments.